packages/components/src/navigator/test/index.tsx

```diff
@@ -108,6 +111,26 @@ function CustomNavigatorGoToBackButton( {
 	);
 }
 
+function CustomNavigatorGoToSkipFocusButton( {
```

Comment:
Interesting — we can keep things as they are for now, but if we find ourselves having to re-create clones of `NavigatorButton` just to pass one specific option, we may consider exposing `options` as a prop.

No action required for now, but something to keep in mind for the future.